features/cellular/framework/AT/AT_CellularDevice.cpp

```diff
@@ -78,50 +73,19 @@ ATHandler *AT_CellularDevice::get_at_handler(FileHandle *fileHandle)
     if (!fileHandle) {
         fileHandle = _fh;
     }
-    ATHandler *atHandler = _atHandlers;
-    while (atHandler) {
-        if (atHandler->get_file_handle() == fileHandle) {
-            atHandler->inc_ref_count();
-            return atHandler;
-        }
-        atHandler = atHandler->_nextATHandler;
-    }
 
-    atHandler = new ATHandler(fileHandle, _queue, _default_timeout, "\r", get_send_delay());
-    if (_modem_debug_on) {
-        atHandler->set_debug(_modem_debug_on);
-    }
-    atHandler->_nextATHandler = _atHandlers;
-    _atHandlers = atHandler;
+    return ATHandler::get(fileHandle, _queue, _default_timeout,
+                          "\r", get_send_delay(), _modem_debug_on);
+}
 
-    return atHandler;
+ATHandler *AT_CellularDevice::get_at_handler()
+{
+    return get_at_handler(NULL);
```

Comment:
What does this do?

Reply:
@kjbracey-arm Through this an ATHandler instance is available for use outside CellularDevice